app/models/miq_user_role.rb

```diff
@@ -22,88 +22,40 @@ class MiqUserRole < ApplicationRecord
   FIXTURE_PATH = File.join(FIXTURE_DIR, table_name)
   FIXTURE_YAML = "#{FIXTURE_PATH}.yml"
 
-  SCOPES = [:base, :one, :sub]
-
   RESTRICTIONS = {
     :user          => "Only User Owned",
     :user_or_group => "Only User or Group Owned"
   }
 
   def feature_identifiers
+    # TODO: Why can't this be #pluck?
     miq_product_features.collect(&:identifier)
   end
 
-  def allows?(options = {})
-    ident = options[:identifier]
-    raise _("No value provided for option :identifier") if ident.nil?
-
-    if ident.kind_of?(MiqProductFeature)
```

Comment:
Note, we dropped support for passing specific `MiqProductFeature` or string identifiers.  Only string identifiers are accepted now.